kolibri/core/auth/management/commands/claim.py

```diff
@@ -0,0 +1,56 @@
+from django.core.management.base import BaseCommand
```

Comment:
Might want to call the management command `register`?

Reply:
Can change all instances of claim to register.